pkg/printers/internalversion/printers.go

```diff
@@ -53,6 +53,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/util/certificate/csr"
 
+	podutil "k8s.io/kubernetes/pkg/api/pod"
```

Comment:
this looks like it will conflict with the next line

Reply:
maybe rename the other podutil to `podutilv1`

Reply:
Thanks for pointing this out. Made the change.